### DISCUSSION TITLE
Quarto Pub account removed, but contents are still there

### DISCUSSION THREAD

**Discussion by jandermoreira** [quarto-dev/quarto-cli#9706] [Q&A] [answered]
### Description

I decided to delete my account at quarto.pub, but after deletion, old contents are still available at [jandermoreira.quarto.pub](http://jandermoreira.quarto.pub).

Now I can't update or delete anything.

Is it a matter of time to the contents to be removed or I need to ask someone for help? In the latter case, who?

**Comment by jandermoreira** [comment]
I tried to recreate my account. I have no access to the old publications and new ones are blocked.

I tried to publish a single page and this is the result.

![image](https://github.com/quarto-dev/quarto-cli/assets/29583273/697a5d09-9e95-4afa-ba7f-f97def92e43a)

I would appreciate if anyone could help me reach the site administrators.

**Comment by softwarenerd** [comment]
@jandermoreira We are investigating the issue.

**Comment by softwarenerd** [comment]
@jandermoreira Please try to recreate your Quarto Pub account and publish to it.

**Reply by jandermoreira** [reply, marked as answer]
Account recreated and woking fine. Old publications are no longer there.
Thank you very much for the quick solution.

**Reply by softwarenerd** [reply]
Great!
